### ISSUE TITLE
Can we check 

### ISSUE THREAD

**Issue by SeminYun** [stgeneral/redmine-progressive-projects-list#92]
We have Versions: ALPHA 3.0 and ALPHA 4.0

ALPHA 3.0 and ALPHA 4.0 are on A and it Shared to B C D

Project Tree
   A  ----B
       ----C
       ----D

We want to show ALPHA 3.0 and 4.0 Progressive bar on B and C and D
But We only show on A

Is it possible?

Our Project Image.  
![image](https://user-images.githubusercontent.com/5058355/97112250-97ecb280-1726-11eb-95c3-c7e7b162bff1.png)

**Comment by stgeneral** [comment]
@SeminYun I don't think so, we use built-in methods to get projects' versions and that's how they look like.
